### PR TITLE
✨ Add new OSS rule

### PR DESCRIPTION
**Pull request by bradystroud**

### CODE REVIEW

**Review by adamcogan — APPROVED** [review]

File: rules/open-source-software/rule.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+authors:
+- title: Brady Stroud
+  url: https://ssw.com.au/people/brady-stroud
+related: []
+redirects: []
+---
+
+Open sourcing software has become increasingly popular for companies as it can help them innovate, attract talent, and build a good reputation. It also helps the community by providing free software that can be used by anyone.
+
+<!-- TODO: Embed video -->
```
Comment (DrLeeHawkins) [inline rules/open-source-software/rule.md:17]: Needs to go after the end intro tag

File: rules/open-source-software/rule.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Some good examples of companies that open source their code are:
+- [RedHat](https://redhatofficial.github.io/)
+- [Microsoft](https://opensource.microsoft.com/)
+- [Google](https://opensource.google/)
+- [Netflix](https://netflix.github.io/)
+- [AirBnB](https://airbnb.io/projects/)
+
+### 🧱 Collaboration
+Open-sourcing opens your project to a broad range of people with unquie skillsets can help accelerate your projects innovation. It also allows you to leverage the work of others to help your project grow.
```
Comment (DrLeeHawkins) [inline rules/open-source-software/rule.md:29]: typo "unquie", missing apostrophe on "projects"

File: rules/open-source-software/rule.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Every time someone submits a PR fixing a bug, you are getting free software development.
+
+### 🔗 Networking
+Being a part of the developer community is a great way to discover potential employees. If a developer has been using a companies open source code for years, they are more likely to want to join and keep using that code.
```
Comment (DrLeeHawkins) [inline rules/open-source-software/rule.md:33]: change "companies" to "company's"

File: rules/open-source-software/rule.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+### 🪧 Demonstrate 
+Making your code public shows confidence in your service and can be a great oppurtunity to demonstrate your companies skills and practices.
```
Comment (DrLeeHawkins) [inline rules/open-source-software/rule.md:36]: change "companies" to "company's"

File: rules/open-source-software/rule.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Every time someone submits a PR fixing a bug, you are getting free software development.
+
+### 🔗 Networking
+Being a part of the developer community is a great way to discover potential employees. If a developer has been using a companies open source code for years, they are more likely to want to join and keep using that code.
+
+### 🪧 Demonstrate 
+Making your code public shows confidence in your service and can be a great oppurtunity to demonstrate your companies skills and practices. 
+
+### 🤝 Sharing solutions
+Open sourcing your code doesn't have to mean making your whole project public. You can take small chunks of code and make them generic enough so its useful to others.
```
Comment (DrLeeHawkins) [inline rules/open-source-software/rule.md:39]: hyphenate "open sourcing" for consistency with other uses in the same rule
change "its" to "they're"

**Review by Hona — APPROVED** [review]

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**